src/components/ViewDeck/ViewDeck.js

```diff
@@ -97,200 +102,136 @@ class ViewDeck extends Component {
-
-    const parsedCards = this.parseCards(includedCards);
-    console.log(
-      "🚀 ~ file: ViewDeck.js ~ line 168 ~ ViewDeck ~ render ~ parsedCards",
-      parsedCards
-    );
+  const getAverage = (costs) => {
+    var total = 0;
+    for (var i = 0; i < costs.length; i++) {
+      total += costs[i];
+    }
+    var avg = total / costs.length;
+    setAverageCmc(avg.toFixed(2));
+  };
 
-    const devotion = this.devotion(parsedCards);
-    console.log(
-      "🚀 ~ file: ViewDeck.js ~ line 204 ~ ViewDeck ~ render ~ devotion",
-      devotion
-    );
+//   console.log(includedCards);
 
-    const cmc = this.cmc(parsedCards);
-    console.log(
-      "🚀 ~ file: ViewDeck.js ~ line 215 ~ ViewDeck ~ render ~ cmc sans land",
-      cmc
-    );
+  return (
+    <div>
+      <h1 className="editDeckHeader">
+        Viewing {selectedDeck.deckname} from {user.username}
+      </h1>
 
-    return (
       <div>
-        <h1 className="editDeckHeader">
-          Viewing {this.props.reduxStore.selectedDeck.deckname} from{" "}
-          {this.props.reduxStore.user.username}{" "}
-        </h1>
-        {/* <h1 id="editDeckHeader">Total Cards: {this.state.deckQty}</h1> */}
-
-        <div className="viewDeckEditDeckView">
-          <div>
-            <img
-              className="featuredViewDeckCard"
-              src={this.state.hoverCard}
-              width="200px"
-              height="280"
-            />
-          </div>
-
-          <br />
-          <h4 className="editDeckHeader">Total Cards: {this.state.deckQty}</h4>
+        <img
+          className="featuredViewDeckCard"
+          src={hoverCard}
+          width="200px"
+          height="280"
+        />
+      </div>
 
-          <table>
-            <thead>
-              <tr>
-                <th></th>
-                {/* Hovercard^ */}
-                <th>Quantity</th>
-                <th className="nameTh">Card Name</th>
-                {/* <th>isCMDR?</th> */}
-                <th></th>
-                {/* isFeatured? */}
-                <th></th>
-                {/* Type */}
-                {/* <th>DELETE</th> */}
+      <div className="ViewDeckMainTable">
+        <br />
+        <h4 className="editDeckHeader">Total Cards: {quantity}</h4>
+
+        <table>
+          <thead>
+            <tr>
+              <th className="nameTh">Quantity</th>
+              <th className="nameTh">Name</th>
+              <th className="typeTh">Type</th>
+              <th>Cost ($)</th>
+            </tr>
+          </thead>
+          <tbody>
+            {includedCards.map((card) => (
+              <tr key={card.id}>
+                <td className="qtyTd">x {card.quantity}</td>
+                <td
+                  className="nameTd"
+                  onMouseOver={() => setDisplayCard(card)}
+                  onMouseLeave={() => removeDisplayCard(card)}
+                >
+                  {card.name}
+                </td>
+                {/* .split("/")[2] */}
```

Comment:
What is this for?

Reply:
I ended up referencing this so I could remove the second part of the type_line from the ViewDeckMainTable.  I think I may have changes in my code I haven't pushed yet, but now that line looks like `                  {card.jsonData.type_line.split("—")[0]}
` 
It's just to remove the subtype from a card if it has one.